addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -320,7 +320,7 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
             --o-wsale-card-thumb-size: calc(160px * var(--o-wsale-card-thumb-aspect-ratio, 1));
 
             h6, .h6 {
-                font-size: $font-size-base * 1.3;
+                @include font-size($font-size-base * 1.3);
```

Comment:
Maybe not supper useful as the rule is already in a media query...

Reply:
Same?